__tests__/app.test.js

```diff
@@ -227,6 +230,7 @@ describe('Test POST /api/article/:article_id/comments endpoint', () => {
         })
```

Comment:
Try to use a more flexible jest matcher here (e.g. .toMatchObject() or .objectContaining()) instead of .toEqual() so you won't have to delete properties. Also further expansion of this endpoint will be possible without failing this test.